src/kOS.Safe/Encapsulation/EnumerableValue.cs

```diff
@@ -1,27 +1,27 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
-using System.Linq;
 using kOS.Safe.Encapsulation.Suffixes;
 using kOS.Safe.Serialization;
+using System.Linq;
 
 namespace kOS.Safe.Encapsulation
 {
-    public abstract class EnumerableValue<T, TC> : Structure, IEnumerable<T>, IDumper where TC : IEnumerable<T>
+    public abstract class EnumerableValue<T, TE> : Structure, IEnumerable<T>, IDumper where TE : IEnumerable<T>
     {
-        protected TC Collection { get; private set; }
+        protected TE Enum { get; private set; }
```

Comment:
I think I'd prefer that this identifier was refactored so it doesn't look like a potential collision with the `enum` type (I know, it doesn't actually conflict with it, I just think it's a bit confusing because of the similarity).  Maybe as simple as changing it to `Enumer`, or `InnerEnumerable`?  Let me know if I sound paranoid about the naming, maybe nothing needs to happen.